services/src/main/java/org/keycloak/services/resources/admin/AdminRoot.java

```diff
@@ -63,6 +63,8 @@
 public class AdminRoot {
     protected static final Logger logger = Logger.getLogger(AdminRoot.class);
 
+    private static final boolean ADMIN_ENABLED = Profile.isFeatureEnabled(Profile.Feature.ADMIN_API);
```

Comment:
Perhaps rename this to ADMIN_API_ENABLED? And also add a ADMIN_UI_ENABLED as well to not have to repeat the check for both ADMIN and ADMIN2 throughout?

Reply:
@pedroigor suggested not to access the Profile in static fields to avoid problems / dependencies with static initialization in quarkus.

Reply:
Perhaps just make the method `public static` in this class considering it belongs here (`AdminRoot` is the JAX-RS resource for the admin console) and should be invoked by other JAX-RS classes?